.travis.yml

```diff
@@ -1,4 +1,12 @@
 language: ruby
 rvm:
-  - 2.1.5
-before_install: gem install bundler -v 1.11.2
+  - 2.3
+  - 2.4
+  - 2.5
+  - 2.6
+
+before_script:
```

Comment:
I think that `bundle` and `bundle exec rake` are the Travis defaults - you might be able to drop both of these sections..